tests/test_security.py

```diff
@@ -1,24 +1,23 @@
-import responses
+from __future__ import annotations
+
 import pytest
+import responses
 
 from pyartifactory import ArtifactorySecurity
-from pyartifactory.models import AuthModel, PasswordModel, ApiKeyModel
-from pyartifactory.exception import InvalidTokenDataException
-
+from pyartifactory.exception import InvalidTokenDataError
+from pyartifactory.models import ApiKeyModel, AuthModel, PasswordModel
 
 URL = "http://localhost:8080/artifactory"
 AUTH = ("user", "password_or_apiKey")
-PASSWORD = PasswordModel(password="test_password")
+PASSWORD = PasswordModel(password="test_password")  # noqa: S106

```

Comment:
Can't `# noqa: S106` be in the pyproject.toml file instead ?

Reply:
done

Reply:
I'm really sorry about that but can you put it back. Indeed, ignoring hardcoded passwords globally is probably a wrong idea since we might miss some issues in the core code

Reply:
here

Reply:
done